core/domain/email_manager.py

```diff
@@ -313,7 +409,18 @@ def get_draft_moderator_action_email(intent):
     """Returns a draft of the text of the body for an email sent immediately
     following a moderator action. An empty body is a signal to the frontend
     that no email will be sent.
+
+    Args:
+        intent: str. The intent string (cause/purpose) of the email.
+
+    Returns:
+        str: draft of the text of the body for an email sent after a moderator
+        action or empty string if no email will be sent.
```

Comment:
Bit hard to parse. Maybe:

str. Draft of the email body for an email sent after a moderator action, or an empty string if no email should be sent.